azurerm/helpers/azure/app_service.go

```diff
@@ -400,7 +400,9 @@ func SchemaAppServiceSiteConfig() *schema.Schema {
 					Optional: true,
 					ValidateFunc: validation.StringInSlice([]string{
 						"2.7",
-						"3.4",
+						"3.6",
```

Comment:
this is a breaking change/will still be supported for existing App Services - so we need to leave this in:
```suggestion
						"3.4",
						"3.6",
```